guides/common/modules/proc_generating-a-list-of-installed-packages.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Generating_a_List_of_Installed_Packages_{context}"]
+= Generating a List of Installed Packages
+
+Use this procedure to generate a list of installed packages in *Report Templates*.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. Select the *Host - All Installed Packages* report template.
+Under the *Preview* tab, the report template shows the host name with the name of the package.
```

Comment:
Is this correct or even necessary to mention? I find this sentence a bit confusing.
Any input @ColeHiggins2 ?

Reply:
Yeah, I'm not sure what this means to be honest. If this is listing the registered hosts then maybe leave it. but I do not think it's necessary. @bangelic can you clarify in comments what this means ?

Reply:
@ColeHiggins2 After our discussion this morning and retesting this against the documentation, I think it's fine to remove it because it probably isn't necessary.